derivation/src/main/scala/caliban/derivation/package.scala

```diff
@@ -0,0 +1,9 @@
+package caliban
+
+import caliban.schema.Schema
+
+import scala.language.experimental.macros
+
+package object derivation {
+  def deriveSchemaInstance[T]: Schema[Any, T] = macro DerivationMacros.deriveSchema[T]
```

Comment:
What happens when a field returns a `ZIO` with R which is not `Any`? We're supposed to get a `Schema[R, *]` but it seems fixed to `Any`.

Reply:
Good catch; the original use case I built this for didn't require this, so it skipped my mind. Will address imminently.